src/components/Documentation/MobileLeftNav.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { docType } from "constants/docType";
+import { MEDIA_QUERIES, PALETTE } from "constants/styles";
+
+import { CloseIcon, Menu } from "basics/Icons";
+
+import { BasicButton } from "basics/Buttons";
+import { Link } from "basics/Links";
+
+import {
+  BetaBadge,
+  NavDivider,
+  NavLogo,
+} from "components/Navigation/SharedStyles";
+import { LeftNav } from "./LeftNav";
+
+const El = styled.div`
+  position: fixed;
+  z-index: 10;
+  @media (${MEDIA_QUERIES.gtTablet}) {
+    display: none;
+  }
+`;
+const NavIconSectionEl = styled.div`
+  display: flex;
+  align-items: center;
+`;
+const NavBarEl = styled.div`
+  display: flex;
+  width: 200vw;
```

Comment:
I feel somewhat clever about this. The menu is actually an overlay that's 2x as wide as the screen. The navbar is 2 navbars next to each other, and when it's open, it moves 1 screen width to the right, exposing the menu.

Reply:
Since it's an overlay, it's maybe a little too clever about pointer events. They're ignored on the topmost element, then un-ignored on the navbar.